src/web.config

```diff
@@ -9,7 +9,7 @@
             <add name="Surrogate-Control" value="no-store" />
             <add name="Pragma" value="no-cache" />
             <add name="Expires" value="0" />
-            <add name="Content-Security-Policy" value="default-src 'self'; font-src 'self' https://fonts.gstatic.com data:; img-src 'self' data: https://www.google-analytics.com https://raw.githubusercontent.com/hmcts; style-src https://fonts.googleapis.com https://fonts.gstatic.com 'unsafe-inline' 'self'; script-src 'self' 'unsafe-inline' 'unsafe-eval' www.google-analytics.com www.googletagmanager.com az416426.vo.msecnd.net; connect-src 'self' *.gov.uk dc.services.visualstudio.com; object-src 'self'; frame-ancestors 'none'" />
+            <add name="Content-Security-Policy" value="default-src 'self'; font-src 'self' https://fonts.gstatic.com data:; img-src 'self' data: https://www.google-analytics.com https://raw.githubusercontent.com/hmcts; style-src https://fonts.googleapis.com https://fonts.gstatic.com 'unsafe-inline' 'self'; script-src 'self' 'unsafe-inline' 'unsafe-eval' www.google-analytics.com www.googletagmanager.com az416426.vo.msecnd.net; connect-src 'self' *.gov.uk dc.services.visualstudio.com *.launchdarkly.com; object-src 'self'; frame-ancestors 'none'" />
```

Comment:
We really don't need anything going into the web.config file anymore, as we don't use IIS/IISNode

Reply:
If anything, this file needs removing

Reply:
@ernestman28  can you confirm on this? I will remove it.